neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -49,6 +49,8 @@ public UInt256 Hash
 
         InventoryType IInventory.InventoryType => InventoryType.TX;
 
+        public bool IsLowPriority => NetworkFee < Settings.Default.LowPriorityThreshold;
```

Comment:
what do you think if we 
`public bool IsLowPriority => NetworkFee < Settings.Default.LowPriorityThreshold || Type == TransactionType.ClaimTransaction;`
and treat all claims as low priority?

Reply:
It is a good idea.

```suggestion
        public bool IsLowPriority => Type == TransactionType.ClaimTransaction || NetworkFee < Settings.Default.LowPriorityThreshold;
```